core-test/src/test/scala/org/bitcoins/core/util/BitcoinSUnitTest.scala

```diff
@@ -0,0 +1,57 @@
+package org.bitcoins.core.util
+
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{FlatSpec, MustMatchers}
+import org.slf4j.LoggerFactory
+
+/** A wrapper for boiler plate testing procesures in bitcoin-s */
+abstract class BitcoinSUnitTest extends FlatSpec with MustMatchers with PropertyChecks {
+
+  lazy val logger = LoggerFactory.getLogger(getClass)
```

Comment:
Don't we have a trait in Bitcoin-S for logging?

Reply:
It is an abstract class, so i'm just going to create a val here for now.